.werft/jobs/build/deploy-to-preview-environment.ts

```diff
@@ -83,8 +83,6 @@ export async function deployToPreviewEnvironment(werft: Werft, jobConfig: JobCon
         | yq r - data['.dockerconfigjson'] \
         | base64 -d)" | base64 -w 0`, { silent: true }).stdout.trim();
 
-    const sweeperImage = exec(`tar xfO /tmp/dev.tar.gz ./sweeper.txt`).stdout.trim();
```

Comment:
Builds on this branch that use job protection will fail because of this line.

Reply:
This can be worked around by running the werft job manually, but instructing it to update the GH status like so

```
werft run github -aupdateGitHubStatus=gitpod-io/gitpod
```

This is what I did here: https://werft.gitpod-dev.com/job/gitpod-build-mads-remove-sweeper.11